app/views/user_mailer/password_reset.text.erb

```diff
@@ -1,2 +1,5 @@
-<%= t "user_mailer.h1" %>
-<%= @greeting %><%= t "user_mailer.p_reset" %>
+<%= t "reset_password.mailer.content_mail.content_1" %>
+
+<%= edit_password_reset_url(@user.reset_token, email: @user.email) %>
```

Comment:
Can I use `_path` instead of `_url`?
Please tell me reason cann't

Reply:
_path helpers provide a site-root-relative path.
_url helpers provide an absolute path, including protocol and server name.
users_url  # => "http://www.example.com/users"
users_path # => "/users"
I can't use _url because I mainly use _url in emails when creating links to the app on the server.

Reply:
I change _url to _path. It not working send mail